src/components/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import logo from '../assets/logo.svg';
+
+const NAV_BAR_ITEMS = ['Catalogue', 'Packaging', 'About Us', 'Contact', 'Eng'];
```

Comment:
this should be an array of objects, each object should have title and href for now